bundle/src/bundle_meta.rs

```diff
@@ -100,6 +108,46 @@ pub struct BundleMetaV0_6_2 {
     pub debug_props: BundleMetaDebugProps,
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq)]
+#[cfg_attr(feature = "pyo3", gen_stub_pyclass, pyclass(get_all))]
+#[cfg_attr(feature = "wasm", derive(Tsify))]
+pub struct BundleMetaV0_6_3 {
+    #[serde(flatten)]
+    pub base_props: BundleMetaBaseProps,
+    #[serde(flatten)]
+    pub junit_props: BundleMetaJunitProps,
+    #[serde(flatten)]
+    pub debug_props: BundleMetaDebugProps,
+    pub bundle_upload_id_v2: Option<String>, // TODO(pat): make this required
```

Comment:
What's preventing us from making it required?

Reply:
It wasn't rolled out to prod yet. Now it is so probably can make this required

Reply:
Done